tests/ui-toml/ifs_same_cond/clippy.toml

```diff
@@ -0,0 +1 @@
+ignore-interior-mutability = ["std::cell::Cell"]
```

Comment:
Could you add a new line at the end of this file?
```suggestion
ignore-interior-mutability = ["std::cell::Cell"]

```

Reply:
Done~